components/basic-transformers/src/main/java/org/datacleaner/beans/numbers/GenerateTimestampTransformer.java

```diff
@@ -19,21 +19,16 @@
  */
 package org.datacleaner.beans.numbers;
 
-import javax.inject.Named;
-
 import org.apache.metamodel.util.HasName;
-import org.datacleaner.api.Categorized;
-import org.datacleaner.api.Configured;
-import org.datacleaner.api.Description;
-import org.datacleaner.api.InputColumn;
-import org.datacleaner.api.InputRow;
-import org.datacleaner.api.OutputColumns;
-import org.datacleaner.api.Transformer;
+import org.datacleaner.api.*;
```

Comment:
Same with the *-ing of a package with many imports. I change my IntelliJ's limit to 30 to not clash with the rest of our team's codestyle.